src/transform/convert/generateSidFields.js

```diff
@@ -26,19 +26,16 @@
 *
 */
 
+import {getHandle} from './utils';
+
 export function generateSID({getFieldValues}, sourceMap) {
   const values = getFieldValues('dc.identifier.uri');
```

Comment:
Source IDs are generated from dc.identifier.uri, which is okay.
Other possible source for source ids might be header...identifier in the OAI-PMH response.